util/src/main/java/io/kubernetes/client/util/ObjectAccessor.java

```diff
@@ -0,0 +1,29 @@
+package io.kubernetes.client.util;
+
+import io.kubernetes.client.openapi.models.V1ObjectMeta;
+import io.kubernetes.client.util.exception.ObjectMetaReflectException;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+/** ObjectAccessor is a set of utils which accesses common fields from objects dynamically */
+public class ObjectAccessor {
+
+  public static final String METHOD_NAME_GET_METADATA = "getMetadata";
```

Comment:
same comment wrt to whether this is needed.

Reply:
updated to private